### PR TITLE
Release 1.12.0

### PR DESCRIPTION
**Pull request by vrubezhny**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
 # Change Log
 
+## 1.12.0 (February 28, 2024)
+ * [#3880](https://github.com/redhat-developer/vscode-openshift-tools/pull/3880) Do not call getAllComponents() in activate.
+ * [#3112](https://github.com/redhat-developer/vscode-openshift-tools/issues/3112) Extension takes several minutes to start if the current `~/.kube/config` context is an inaccessible cluster.
+ * [#3931](https://github.com/redhat-developer/vscode-openshift-tools/issues/3931) New Serverless Function is not shown after creation.
+ * [#3928](https://github.com/redhat-developer/vscode-openshift-tools/issues/3928) Can't load Helm chart.
+ * [#3879](https://github.com/redhat-developer/vscode-openshift-tools/pull/3879) When turned to using service account for a Sandbox a wrong warning me….
```
Comment (mohitsuman) [inline CHANGELOG.md:8]:
```suggestion
 * [#3879](https://github.com/redhat-developer/vscode-openshift-tools/pull/3879) When turned to use a service account for a Sandbox a wrong warning message appears.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+ * [#3848](https://github.com/redhat-developer/vscode-openshift-tools/pull/3848) Stop trying to connect to a cluster when it doesn't appear to be avai….
```
Comment (mohitsuman) [inline CHANGELOG.md:9]:
```suggestion
 * [#3848](https://github.com/redhat-developer/vscode-openshift-tools/pull/3848) Stop trying to connect to a cluster when it doesn't appear to be available.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+ * [#3841](https://github.com/redhat-developer/vscode-openshift-tools/pull/3841) Update func to 1.13.0.
```
Comment (mohitsuman) [inline CHANGELOG.md:10]:
```suggestion
 * [#3841](https://github.com/redhat-developer/vscode-openshift-tools/pull/3841) Update func CLI to 1.13.0.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,27 @@
...
+ * [#3838](https://github.com/redhat-developer/vscode-openshift-tools/issues/3838) Developer Sandbox Login experience enhancement.
```
Comment (mohitsuman) [inline CHANGELOG.md:11]:
```suggestion
 * [#3838](https://github.com/redhat-developer/vscode-openshift-tools/issues/3838) New and Improved sign-in for Developer Sandbox Login experience.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,27 @@
...
+ * [#3822](https://github.com/redhat-developer/vscode-openshift-tools/issues/3822) Context menu item to scale Deployments.
```
Comment (mohitsuman) [inline CHANGELOG.md:12]:
```suggestion
 * [#3822](https://github.com/redhat-developer/vscode-openshift-tools/issues/3822) Add Context menu item to scale OpenShift Deployments.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,27 @@
...
+ * [#3814](https://github.com/redhat-developer/vscode-openshift-tools/issues/3814) "Open in Console" context menu item appears for non-OpenShift clusters.
+ * [#3813](https://github.com/redhat-developer/vscode-openshift-tools/pull/3813) Force refresh App explorer on create project....
+ * [#3812](https://github.com/redhat-developer/vscode-openshift-tools/issues/3812) As a user I initially couldn't find "Change Active Project".
+ * [#3804](https://github.com/redhat-developer/vscode-openshift-tools/pull/3804) Improve switch context workflow.
+ * [#3665](https://github.com/redhat-developer/vscode-openshift-tools/issues/3665) Investigate replacing `Odo.isPodmanPresent`.
```
Comment (mohitsuman) [inline CHANGELOG.md:17]:
```suggestion
 * [#3665](https://github.com/redhat-developer/vscode-openshift-tools/issues/3665) Enhance dev on podman experience
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,27 @@
...
+ * [#3640](https://github.com/redhat-developer/vscode-openshift-tools/issues/3640) Show context menu to Open in Terminal for the function created.
+ * [#3638](https://github.com/redhat-developer/vscode-openshift-tools/issues/3638) Show vscode notification message if docker is not started for Fn commands.
+ * [#3512](https://github.com/redhat-developer/vscode-openshift-tools/issues/3512) Add paste to the OpenShift terminal.
+ * [#3405](https://github.com/redhat-developer/vscode-openshift-tools/issues/3405) podman installed but propose to install podman.
```
Comment (mohitsuman) [inline CHANGELOG.md:21]:
```suggestion
 * [#3405](https://github.com/redhat-developer/vscode-openshift-tools/issues/3405) Fix podman detection prompt
```

**Review by mohitsuman — CHANGES_REQUESTED** [review]
added some wording changes

File: CHANGELOG.md (hunk 1 above)
Comment (vrubezhny) [reply CHANGELOG.md]: Done
Plus  one  more commit title is extracted:

```
- * [#3813](https://github.com/redhat-developer/vscode-openshift-tools/pull/3813) Force refresh App explorer on create project....
+ * [#3813](https://github.com/redhat-developer/vscode-openshift-tools/pull/3813) Force refresh App explorer on create project when new project name is the same as the one set in the current-context.
```

File: CHANGELOG.md (hunk 2 above)
Comment (vrubezhny) [reply CHANGELOG.md:9]: Done

File: CHANGELOG.md (hunk 3 above)
Comment (vrubezhny) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 4 above)
Comment (vrubezhny) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 5 above)
Comment (vrubezhny) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 6 above)
Comment (vrubezhny) [reply CHANGELOG.md]: Done

**Comment by vrubezhny** [comment]
> added some wording changes

@mohitsuman I've added the suggested changes

**Review by mohitsuman — APPROVED** [review]
lgtm

**Pull request merged**